src/utils/ButtonWithIcon.vue

```diff
@@ -128,9 +128,9 @@ export default {
         labelClasses() {
             let className
             if (this.direction === 'column') {
-                className = this.iconsBeforeText ? 'mt-1' : 'mb-1'
+                className = this.iconsBeforeText ? 'mt-2' : 'mb-2'
             } else {
-                className = this.iconsBeforeText ? 'ms-1' : 'me-1'
+                className = this.iconsBeforeText ? 'ms-2' : 'me-2'
```

Comment:
I have the impression that this pushes the text a bit off center, we should either compensate on the other side or go back to a smaller margin.
What was your goal by setting a bigger margin here?

Reply:
The goal is to have a margin between icon and label, not have them too close
Before:
![image](https://user-images.githubusercontent.com/67745584/200794331-12f52988-c4d2-4ad4-868a-b233d46a0008.png)
After:
![image](https://user-images.githubusercontent.com/67745584/200794467-05719648-7353-4098-bd5a-7363e0ee1384.png)

The issue also mentioned by @jedef is that the `ButtonWithIcon` class can be used without icon ! It is not very user friendly. I'll quickly fix this but we should consider some rework here to have the class either renamed and only used with icon. IMHO a button without icon doesn't require a component as we use bootstrap.

Reply:
By the way this of center was already present, just less bigger as it used a `me-1` instead of `me-2`.

Reply:
the goal having a single component to manage the button was specifically to encapsulate bootstrap usage in one place, so that if we wanted to switch to something else, or tweak the design of existing buttons, it would only need a change in one place.